src/rs.cpp

```diff
@@ -136,6 +141,29 @@ struct rs2_sensor : public rs2_options
     rsutils::subscription subscription;
 };
 
+namespace librealsense {
+
+template<>
+struct arg_streamer< rs2_options *, false >
+{
+    void stream_arg( std::ostream & out, rs2_options * options, bool last )
```

Comment:
Is this used? Couldn't find where

Reply:
It was used. I removed for now. Let's talk about it if you're curious...